src/mudlet-lua/lua/Other.lua

```diff
@@ -832,3 +806,66 @@ do
     end
   end
 end
+
+local timeframetable = {}
+
+function timeframe(vname, true_time, nil_time, ...)
+  assert(type(true_time) == "number" or type(true_time) == "table", "Timeframe requires a number or table argument as first parameter")
+  assert(type(nil_time) == "nil" or type(nil_time) == "number" or type(nil_time) == "table", "Timeframe requires a number or table if providing a second parameter")
```

Comment:
Can you change these 2 messages to conform to our (newer) standard `<function>: bad argument #<arg#> type (<param meaning> as <type(s)> expected, got %s!` with `%s` being the actual type (`string.format` replaced)?